operator/config/crd/bases/core.paladin.io_paladins.yaml

```diff
@@ -68,10 +68,68 @@ spec:
                 required:
                 - authMethod
                 type: object
+              baseLedgerEndpoint:
+                description: BaseLedgerEndpoint specifies the base endpoint for the
+                  ledger
+                properties:
+                  endpoint:
+                    description: Network specifies the configuration when the type
+                      is 'network'.
+                    properties:
+                      authConfig:
```

Comment:
the `Config` part of this field name seems redundant.  Should it just be `auth` ?